{
	"@context": "https://schema.org",
	"@type": "Article",
	"publisher": {
		"@type": "Organization",
		"name": "The Cloudflare Blog",
		"logo": {
			"@type": "ImageObject",
			"url": "https://blog-cloudflare-com-assets.storage.googleapis.com/2019/06/logo-cloudflare-dark-1.svg",
			"width": 109,
			"height": 40.5
		}
	},
	"author": {
		"@type": "Person",
		"name": "Gabriel Gabor",
		"image": {
			"@type": "ImageObject",
			"url": "http://blog.cloudflare.com/content/images/2021/03/unnamed.jpg",
			"width": 600,
			"height": 600
		},
		"url": "https://blog.cloudflare.com/author/gabriel/",
		"sameAs": []
	},
	"headline": "防御另一个 Log4j RCE 漏洞 CVE-2021-45046",
	"url": "https://blog.cloudflare.com/zh-cn/protection-against-cve-2021-45046-the-additional-log4j-rce-vulnerability-zh-cn/",
	"datePublished": "2021-12-15T13:56:00.000Z",
	"dateModified": "2021-12-28T22:50:25.000Z",
	"image": {
		"@type": "ImageObject",
		"url": "http://blog.cloudflare.com/content/images/2021/12/image1-80-3.png",
		"width": 1200,
		"height": 628
	},
	"keywords": "Log4J (CN), WAF (CN), Log4Shell (CN), 漏洞 (CN), 简体中文 (CN)",
	"description": "继 CVE-2021-44228 之后，已提交了第二个 Log4J CVE：CVE-2021-45046。我们之前针对 CVE-2021-44228 发布的规则针对这个新的 CVE 提供相同级别的保护",
	"mainEntityOfPage": {
		"@type": "WebPage",
		"@id": "http://blog.cloudflare.com/"
	}
}